indexer/drawing_rule_def.hpp

```diff
@@ -2,9 +2,21 @@
 
 #include "base/buffer_vector.hpp"
 
+#include "drape/utils/projection.hpp"
 
 namespace drule
 {
+
+// Priority range for area and line drules. Should be same as PRIORITY_RANGE in kothic.
+// Each layer = +/-1 value shifts the range by this number, so that e.g. priorities
+// of the default layer=0 range [0;1000) don't intersect with layer=-1 range [-1000;0) and so on..
+double constexpr kLayerDepthRange = 1000;
```

Comment:
Controversial why _depth_ constant is in rules.

Reply:
because its both *depth* and *priority* range at the same time;
could be renamed to e.g. `kPriorityRange`, but then it'll look controversial in stylist which operates with depths :)

Reply:
I don't see a reason to move this constant here. Just revert like it was before?